scalar/in.go

```diff
@@ -24,9 +29,9 @@ func InStrings(subject string, collection []string) bool {
 	return false
 }
 
-// InStringPs returns true if the subject string is contained in the supplied
-// slice of string pointers
-func InStringPs(subject string, collection []*string) bool {
+// PtrsContains returns true if the subject scalar is contained in the supplied
+// slice of scalar pointers
+func PtrsContains[s Scalar](collection []*s, subject s) bool {
```

Comment:
Both of these functions can be replaced with `samber/lo.Contains`:

https://github.com/samber/lo/blob/a0c1ea371a527dadf4957b88e871db4becf54d43/intersect.go#L4

Reply:
+1 Let's not reinvent the wheel here

Reply:
Getting rid of this file